code/game/gamemodes/blob/powers.dm

```diff
@@ -95,12 +95,13 @@
 	playsound(B.loc, 'sound/effects/splat.ogg', 50, 1)
 	blobber.overmind = src
 	blobber.update_icons()
-	blobber.AIStatus = AI_OFF
+	blobber.notransform = 1 //stop the naut from moving around
```

Comment:
notransform (formally known as monkeyizing) is a constant victim of this sort of thing. Shit is just to good.

Lets rename it to "hold_still_and_don't_get_screwed_with".